.github/workflows/lint.yml

```diff
@@ -43,3 +49,33 @@ jobs:
           echo "cd serve/"
           echo "pre-commit install"
           echo "pre-commit will lint your code for you, so git add and commit those new changes and this check should become green"
+  spellcheck:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Install dependencies
+        run: |
+          sudo apt-get install aspell aspell-en
+          pip install pyspelling
+
+      - name: Get changed files
+        id: changed-files
+        uses: tj-actions/changed-files@v29.0.4
+        with:
+          files: | 
+            **/*.md
+
+      - name: Check spellings
+        run: |
+          sources=""
+          for file in ${{ steps.changed-files.outputs.all_changed_files }}; do
+            sources+=" -S $file"
+          done
+          pyspelling -c $GITHUB_WORKSPACE/ts_scripts/spellcheck_conf/spellcheck.yaml --name Markdown $sources
```

Comment:
[wordlist.dic](https://github.com/pytorch/serve/blob/master/ts_scripts/spellcheck_conf/wordlist.dic) is currently tracked by git. Since we've updated the `wordlist.txt` file, should we include the updated `wordlist.dic` file as well in this commit? If it is auto-generated each time `pyspelling` is run, we could `git rm` the file and not track it in future by including it in `.gitignore`

Reply:
I agree. I do it